client/src/components/home/Home.tsx

```diff
@@ -0,0 +1,141 @@
+            <Text styles={font16pxStyle}>Frequently asked questions</Text>
+            <Stack styles={btnStackStyles}>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    How do I change my Microsoft Bookings page URL?
+                  </StackItem>
+                  <StackItem align="center" style={textDecorationNone}>
+                    <span style={newWindowIconWrapper}>
+                      <Icon iconName="OpenInNewWindow" />
+                    </span>
+                  </StackItem>
+                </Stack>
+              </Link>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    How do I switch from using Rooms (preview) to using Microsoft Bookings?
+                  </StackItem>
+                  <StackItem align="center" style={textDecorationNone}>
+                    <span style={newWindowIconWrapper}>
+                      <Icon iconName="OpenInNewWindow" />
+                    </span>
+                  </StackItem>
+                </Stack>
+              </Link>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    More frequently asked questions
+                  </StackItem>
+                  <StackItem align="center" style={textDecorationNone}>
+                    <span style={newWindowIconWrapper}>
+                      <Icon iconName="OpenInNewWindow" />
+                    </span>
+                  </StackItem>
+                </Stack>
+              </Link>
+            </Stack>
+            <Text styles={font16pxStyle}>Learn more about Azure Communication Services</Text>
+            <Link styles={font12pxSemiBoldStyle}>Azure Communication Services virtual appointments</Link>
+            <Text styles={font12pxWeight400Style}>
+              This tutorial describes concepts for virtual visit applications. After completing this tutorial and the
+              associated Sample Builder, you will understand common use cases that a virtual appointments application
+              delivers
+            </Text>
+            <Link styles={font12pxSemiBoldStyle}>Azure Communication Services Rooms (Preview)</Link>
+            <Text styles={font12pxWeight400Style}>
+              Azure Communication Services provides a concept of a room for developers who are building structured
+              conversations such as virtual appointments or virtual events. Rooms currently allow voice and video
+              delivers
+            </Text>
+            <Link styles={font12pxSemiBoldStyle}>Get Started with UI Library</Link>
```

Comment:
Looks like these `Link`s are missing hrefs. Are these WIP?

Reply:
yes this is wip